src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -1645,7 +1635,7 @@ export class StudyViewPageStore {
         onError: (error => {}),
         onResult: (data=>{
             data.forEach(item => {
-                const uniqueKey = getClinicalAttributeUniqueKeyByDataTypeAttrId(item.clinicalDataType, item.attributeId);
+                const uniqueKey = item.attributeId;
```

Comment:
Let's use the method getClinicalAttributeUniqueKey

Reply:
getClinicalAttributeUniqueKey accepts ClinicalAttribute, which is not possible here